var/spack/repos/builtin/packages/ucx/package.py

```diff
@@ -198,11 +198,18 @@ def configure_args(self):
         args += self.with_or_without("gdrcopy", activation_value="prefix")
         args += self.with_or_without("ib-hw-tm", variant="ib_hw_tm")
         args += self.with_or_without("knem", activation_value="prefix")
-        args += self.with_or_without("mlx5-dv", variant="mlx5_dv")
         args += self.with_or_without("rc")
         args += self.with_or_without("ud")
         args += self.with_or_without("xpmem", activation_value="prefix")
 
+        # mlx5_dv
+        # UCX <= 1.17: --with-mlx5-dv
+        # UCX >= 1.18: --with-mlx5
+        if "@:1.17" in spec:
```

Comment:
```suggestion
        if spec.satisfies("@:1.17"):
```

It's now recommended to use `spec.satisfies()` instead of `in`.

Reply:
thanks.  pushed change